ci/gha-win-env.yml

```
@@ -8,13 +8,12 @@ dependencies:
   - cmake
   - git
   - m2w64-htslib
-  - tiledb=2.15
+  - tiledb=2.25
   - vs2019_win-64
```

```suggestion
  - vs2022_win-64
```

I doubt this will fix it since [CMake](https://github.com/TileDB-Inc/TileDB-VCF/actions/runs/10625109379/job/29454539691?pr=762#step:4:183) already finds Visual Studio 2022:

```
-- Check for working CXX compiler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.29.30133/bin/HostX64/x64/cl.exe - skipped
```

But worth a try just in case it helps to have the conda-installed wrapper match. We updated to Visual Studio 2022 when tiledb-feedstock was migrated to 2.22 (https://github.com/conda-forge/tiledb-feedstock/pull/275)

---

@gspowley Thanks for trying this out! Unfortunately it still [failed](https://github.com/TileDB-Inc/TileDB-VCF/actions/runs/10793027676) with the same errors, but at least we ruled out this simple intervention